### PR TITLE
[2/Q] Edit volunteering opportunity and get volunteering opportunity [Back-end]

### PR DESCRIPTION
**Pull request by nikhitashanker**
- Add servlet to get volunteering opportunities based on a given eventId
- Add servlet to get single volunteering opportunity based on given opportunityId
- Add update functionality to form handler servlet
- Opportunity parameters for functions naming made consistent

### CODE REVIEW

File: project/src/main/java/com/google/sps/servlets/VolunteeringOpportunityDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
```
Comment (liangqigong) [inline project/src/main/java/com/google/sps/servlets/VolunteeringOpportunityDataServlet.java:1]: Can you put this file in a different PR? I feel there are 2 features in the PR.

File: project/src/main/java/com/google/sps/servlets/EventVolunteeringDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
```
Comment (liangqigong) [inline project/src/main/java/com/google/sps/servlets/EventVolunteeringDataServlet.java:1]: And this file should be in another PR. So every PR can be small, in order to be reviewed fast and merged fast.

File: project/src/main/java/com/google/sps/servlets/VolunteeringOpportunityDataServlet.java (hunk 1 above)
Comment (nikhitashanker) [reply project/src/main/java/com/google/sps/servlets/VolunteeringOpportunityDataServlet.java:1]: Done - I have separated this PR into 3 PRs 1 per feature. Thanks.

File: project/src/main/java/com/google/sps/servlets/EventVolunteeringDataServlet.java (hunk 2 above)
Comment (nikhitashanker) [reply project/src/main/java/com/google/sps/servlets/EventVolunteeringDataServlet.java:1]: Done - Yes, I think it is organized better now, thanks.

**Comment by nikhitashanker** [comment]
Closed and separated into 3 separate smaller PRs.
